Mainframe3270/x3270.py

```diff
@@ -55,6 +59,8 @@ def open_connection(
         host: str,
         LU: Optional[str] = None,
         port: int = 23,
+        height: Optional[int] = 24,
```

Comment:
Actually, I think it is cleaner to leave the `Open Connection` keyword as is. The model should be determined beforehand, either from the import, or if a user really goes to that length, from the `extra_args` parameter.

Here, please bear in mind that the library scope is TEST SUITE, i.e. an instance of the Library is created for each suite. This mean, when the model is specified on the import level, it will be that model for all Open Connection calls.

However, if a user chooses to specify the model in Open Connection, the model should just be used in this connection. If Open Connection is called without extra_args later in the test suite and without extra_args, it should take the model from the import.